composer.json

```
@@ -1,21 +1,21 @@
 {
-  "name":         "egulias/email-validator",
-  "description":  "A library for validating emails against several RFCs",
-  "homepage":     "https://github.com/egulias/EmailValidator",
-  "keywords":     ["email", "validation", "validator", "emailvalidation", "emailvalidator"],
-  "license":      "MIT",
+  "name":        "egulias/email-validator",
+  "description": "A library for validating emails against several RFCs",
+  "homepage":    "https://github.com/egulias/EmailValidator",
+  "keywords":    ["email", "validation", "validator", "emailvalidation", "emailvalidator"],
+  "license":     "MIT",
   "authors": [
     {"name": "Eduardo Gulias Davis"}
   ],
   "extra": {
     "branch-alias": {
-      "dev-master": "3.0.x-dev"
+      "dev-master": "4.0.x-dev"
     }
   },
   "require": {
-    "php": ">=7.2",
-    "doctrine/lexer": "^1.2",
-    "symfony/polyfill-intl-idn": "^1.15"
+    "php": ">=8.1",
+    "doctrine/lexer": "^3.0",
```

Again, I would strongly recommend to loosen this constraint to…

```suggestion
    "doctrine/lexer": "^2.0 || ^3.0",
```

This should be possible without much extra effort and avoid dependency conflicts downstream for the time being.

---

I have checked Doctrine's ORM and v3 is requiring v2 of the lexer, so @Ph0tonic can you please update to 
```suggestion
    "doctrine/lexer": "^2 || ^3",
```
Thanks @derrabus

---

Done 👍